contracts/Clearable.sol

```diff
@@ -0,0 +1,159 @@
+pragma solidity ^0.5.0;
+
+import "eip1996/contracts/Holdable.sol";
+import "./IClearable.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+
+contract Clearable is Holdable, IClearable, Ownable {
+
+    using StringUtil for string;
+
+    struct ClearableTransfer {
+        address orderer;
```

Comment:
I would only save the things in the struct, that are not already saved in the hold. If not we pay more gas, because we save things more than once.
In this case only the status should be needed, because it is different than in the hold.

Reply:
Ok makes sense!